distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestSecurity.java

```diff
@@ -309,11 +343,26 @@
     public static final String SAMPLE_SOAP =
             "<?xml version=\"1.0\"?><soap:Envelope xmlns:soap=\"http://schemas.xmlsoap.org/soap/envelope/\"><soap:Body><helloWorld xmlns=\"http://ddf.sdk/soap/hello\" /></soap:Body></soap:Envelope>";
 
+    @Override
+    protected Option[] configureSystemSettings() {
```

Comment:
Unfortunately doesn't look like that'll be a smooth transition, I'll figure it out